packages/core-react/src/context/reefKnot.tsx

```diff
@@ -9,7 +9,8 @@ export interface ReefKnotContextProps {
   rpc: Record<number, string>;
   walletconnectProjectId?: string;
   chains: Chain[];
-  defaultChain?: Chain;
+  defaultChain: Chain;
+  children?: ReactNode;
```

Comment:
Use `React.PropsWithChildren<propsType>` for best typing. ReactNode does not cover all children edgecases.

Reply:
`type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };`
So basically, it is almost the same, but more verbose.
Also, it makes `children` optional, which is okay here, but not okay in some other cases